src/services/comparer.ts

```diff
@@ -26,6 +31,22 @@ export async function chooseFoldersAndCompare(path?: string) {
   return compareFolders();
 }
 
+async function showDiff(uri1: Uri, uri2: Uri, title: string): Promise<void> {
```

Comment:
Let's have a meaningful name to express the different between `showDiff` and `showDiffs`

```suggestion
async function openDiffView(uri1: Uri, uri2: Uri, title: string): Promise<void> {
```